gr-blocks/grc/blocks_sigmf_sink_minimal.block.yml

```diff
@@ -0,0 +1,62 @@
+id: blocks_sigmf_sink_minimal
+label: SigMF Sink (Minimal)
+flags: [ python, deprecated ]
+
+parameters:
+-   id: filename
+    label: File Name
+    dtype: file_save
+-   id: type
+    label: Stream Type
+    dtype: enum
+    options: [complex float (fc32_le), real float (rc32_le), complex short (sc16_le), real short (rc16_le)]
+    option_attributes:
+        type: [complex, float, short, short]
+        size: [gr.sizeof_gr_complex, gr.sizeof_float, gr.sizeof_short, gr.sizeof_short]
```

Comment:
Is complex short and real short the same here?

Reply:
They actually are since the shorts are interleaved but come one part per item. At some point we might consider making a special type for this to avoid this situation.

Reply:
Yep the only difference is how it gets annotated in the metafile, and there's no way for the block to know which one it is unless you tell it, so that's what the ishort param is for.

Reply:
IShort is one of those datatypes that are confusing. They exist somehow but are poorly supported. VOLK defines smth similar to `std::complex<short>` but it seems like it only exists for the sake of completeness. i.e. it is mostly unsupported.

Reply:
As far as I know it exists primarily as a file format; half the size of fc32 yet has enough dynamic range to support a lot of RF use cases.